app/javascript/channels/instance_channel.js

```diff
@@ -0,0 +1,26 @@
+import consumer from "./consumer";
+
+const initInstanceChannel = () => {
+  console.log("Connection complete");
+  const instanceContainer = document.getElementById("instance")
+  if (instanceContainer) {
+    const id = instanceContainer.dataset.instanceId;
+    console.log("Instance id: ", id);
```

Comment:
In order to know if the user has successfully subscribed to the channel, I think this line should be inside `connected() {` from line 11.